ui/src/tracks/async_slices/index.ts

```diff
@@ -137,6 +143,33 @@ export class AsyncSliceTrack extends ChromeSliceTrack {
   static create(args: NewTrackArgs): Track {
     return new AsyncSliceTrack(args);
   }
+
+  drawChevron(ctx: CanvasRenderingContext2D, instant?: Instant) {
```

Comment:
Renaming the method isn't desirable because of the deltas it would create with upstream, but please mitigate the situation with a comment that reveals it draws a diamond if a suitable [instant] is passed in.